lms/templates/login.html

```diff
@@ -27,6 +27,13 @@ <h1 id="main-title" class="h3 font-weight-normal">{{ _('התחברות') }}</h1>
           <input class="form-control form-control-lg" type="hidden" name="next" id="next" value="{{ request.args.get('next', '') }}">
           <button class="btn btn-primary btn-lg btn-block">{{ _('התחבר') }}</button>
         </form>
+        {% if login_error %}
```

Comment:
Looking at different [error messages](https://www.google.com/search?q=login+error&tbm=isch) across the web, it's usually just below the login title or near the faulty field. I think we should follow this pattern.

Reply:
good point, I put it under the login